packages/main/src/plugin/extension-loader.ts

```diff
@@ -597,6 +599,20 @@ export class ExtensionLoader {
       },
     };
 
+    const authentticationProviderRegistry = this.authentticationProviderRegistry;
```

Comment:
you've a small typo here:

```suggestion
    const authenticationProviderRegistry = this.authenticationProviderRegistry;
```